LocalSettings.php

```diff
@@ -3290,6 +3291,7 @@
 				'autopatrolled',
 				'rollback',
 				'bot',
+				'sysop',
```

Comment:
the entire crat section is uneeded i think since crats already have this (and this array is just adding to existing permissions...)